requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
 docker>=3.7.0,<3.8.0
-exam>=0.5.1
+exam>=0.10.6
```

Comment:
Any specific reasons for bumping?

Reply:
it's hidden in the long commit log, i'll probably separate that from this pr

Reply:
https://github.com/getsentry/sentry/pull/35732